api/src/services/script-runner.service.ts

```diff
@@ -1281,6 +1281,72 @@ export class ScriptRunnerService {
     return { success: true };
   }
 
+  async removeWorkAddresses(req: ExpressRequest): Promise<SuccessDTO> {
+    const requestingUser = mapTo(User, req['user']);
+    await this.markScriptAsRunStart('remove work addresses', requestingUser);
+
+    const applicants = await this.prisma.applicant.findMany({
```

Comment:
**issue:** I think its best to skip this read. Depending on how many applications are in doorway this could cause us some mem issues